src/Components/RecommendationsPage/RecommendationsPage.scss

```diff
@@ -0,0 +1,3 @@
+a {
```

Comment:
?

Reply:
hmm, the links were black without this for some reason, now its not the case somehow, removing...